### DISCUSSION TITLE
Content Editor - The SAVE button does not get activated once you upload the audio to a stage

### DISCUSSION THREAD

**Discussion by vinod303** [Sunbird-Ed/Community#26] [Bugs] [answered]
There seems to be an issue in the Content Editor while uploading / adding an audio to a stage.

**Here is the scenario:**

1. Create/Edit a resource
2. add a slide
3.  Insert an image or video/text in the slide
4.  Save the slide by clicking on the SAVE button("SAVE" button is enabled now)
5.  Then, Add an audio to the slide (either select an existing one or upload a new one and add)
6.  Notice that, the "SAVE" button still  in INACTIVE state. ie. the user is not able to click on SAVE to save the latest update to the content.

Unless the user do some/any small change on the slide(after CLICKING on SAVE), the SAVE is not getting Enabled for the user to click when upload an audio.

**Comment by luckynvdu** [comment, marked as answer]
@vinukumar-vs can you please look into this?

cc @rhwarrier

**Reply by vinukumar-vs** [reply]
[vinod303](https://github.com/vinod303) This is the existing issue. Until any changes made to the slide, save button will not enable. Please suggest the same workaround as you mentioned to make a small changes to any slide. So that save button will get enabled. 

We are not making any changes to the existing interactive editor until it is a very critical issue. 

@luckynvdu

**Reply by sharathkashyap** [reply]
@vinod303 @vinukumar-vs Locking this thread due to inactivity.

@luckynvdu
